src/frontend/screens/Settings/components/SettingsModal/index.scss

```diff
@@ -2,8 +2,8 @@
   width: 65vw;
   display: flex;
   flex-direction: column;
-  justify-content: space-between;
   max-width: 800px;
+  min-height: 50vh;
```

Comment:
maybe we can also position the modal a fixed amount from the top? now changing tabs changes the height and it keeps jumping around, it's not great for accessibility that much movement

Reply:
good idea, that was bothering me as well. I will try.

Reply:
done!